modules/exploits/windows/smtp/sysgauge_client_bof.rb

```diff
@@ -0,0 +1,89 @@
+
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+#
+# Fuzzer written by corelanc0d3r - <peter.ve [at] corelan.be>
+# http://www.corelan.be:8800/index.php/2010/10/12/death-of-an-ftp-client/
+#
+##
+
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Exploit::Remote
+  include Msf::Exploit::Remote::TcpServer
+
+  Rank = NormalRanking
+
+  def initialize()
+    super(
+      'Name'           => 'SysGauge SMTP Validation Buffer Overflow',
+      'Description'    => %q{
+        This module will setup a SMTP server expecting a connection from SysGauge 1.5.18
+        via its SMTP server validation. The module sends a malicious response along in the
+        220 service ready response and exploits the client resulting in an unprivileged shell.
+      },
+      'Author'         =>
+      [
+        'Chris Higgins', # msf Module -- @ch1gg1ns
+        'Peter Baris'
```

Comment:
You should specify that Peter Baris wrote the PoC. It appears they found the vuln, too?